### PR TITLE
Feature/readme

### LINKED ISSUE #2: Add PR Template
- [x] Add PR Template 
- [x] Test PR Template 

image

### PR DESCRIPTION
**Pull request by hayleyw7**
# Description drawing

Expanded README.md and added more images

#10 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [ ] local host 3000
- [ ] console.log()
- [ ] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
-- PoKedeX desktop: when I visit the app, all PoKemon should be displayed.
-- Home-Page: as a user, I should be able to access a page about how to use the app.
-- Selected PoKemon: As a user, I can click a card, and see that Pokemon’s details. When a PoKemon’s details are displayed, none of the other PoKemon cards will be visible.
-- When a PoKemon’s details are displayed, the user should have a way to return to the main view of all movies.
+- PoKedeX: As a user, when I visit the app, I should be able to search for Pokemon by name.
+- PoKedeX: As a user, when I visit the app on a computer or tablet, I should be able to see a display of all Pokemon.
+- Help: As a user, when I visit the app, I have a "help" button, and if I click it, I can access a page that explains how to use the app.
+- PokemonDetails: As a user, I should be able to access a page that contains a Pokemon's details by either a) clicking their image or b) searching for their name, and no other Pokemon details or images should be visible.
+- PokemonDetails & Help: As a user, I should be able to return to the PoKedeX from any other page.
```
Comment (joanafbrito) [inline README.md:13]: I really like the changes here! Great use of the user story!

File: README.md [hunk 2]
```diff
@@ -1,28 +1,31 @@
...
 
-## A Little bit about the Developer
+This project was a group project for Turing School of Software & Design during module 3. [The project rubric is linked here.](https://frontend.turing.edu/projects/module-3/stretch.html)
+
+
+## A Little Bit About the Developers
+<img src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/25.svg" alt="Pikachu" width="100" align="right"/>
```
Comment (joanafbrito) [inline README.md:20]: Thank your for showing us how to style the image in a ` .md ` format!

File: README.md [hunk 3]
```diff
@@ -1,28 +1,31 @@
...
 
 *Hi! My name is [Joana](https://github.com/joanafbrito) and I am a current Front End Engineering student at Turing School of Software Design. I am a career changer! I've  spent the last decade working to manage two small business, and grabbing the opportunity to move to another country.* 
 *Now, I'm spending my days learning code. As a woman I can contribuit to build a more balanced society, where people can be seen as equals in the software marketing and where I can build apps more inclusive and make a difference using this huge platform of the internet!*
 
+*Info about [Hayley](https://github.com/hayleyw7)🐈 *
```
Comment (joanafbrito) [inline README.md:24]: We should remember to add yours and Matt's paragraph !

File: README.md [hunk 4]
```diff
@@ -51,54 +51,48 @@
...
 
 ## Future Designs/Extensions
 
-- Add a user login/logout with username and password .
-- Have an area of favorited cards.
-- Be able to play the game in the app.
-- The User's favotiting will be saved within the user's account.
+- The user will be able to login/logout with username and password.
+- The user will be able to save their favorite cards within their account.
+- The user will be able to play the game in the app.
```
Comment (joanafbrito) [inline README.md:98]: Again, great use of the user story ! Love it ! Thank you!

**Review by joanafbrito — APPROVED** [review]
Overall I really like the use of the **_user story_** in the readme, I definitely learned from that 
Styling an `img` on `md `it is a plus  !

- For next steps we should add the paragraph about you and about Matt

File: README.md [hunk 5]
```diff
@@ -51,54 +51,48 @@
...
+- Managing the media queries for responsive design
+- Learning and using two new tecnologies at the same time (GSAP and PWA)
 
 ## Future Designs/Extensions
 
-- Add a user login/logout with username and password .
-- Have an area of favorited cards.
-- Be able to play the game in the app.
-- The User's favotiting will be saved within the user's account.
+- The user will be able to login/logout with username and password.
```
Comment (Matt-Roden) [inline README.md:96]: Love the user story, I think it gives the read me a clear description of what the app is!

File: README.md [hunk 6]
```diff
@@ -1,28 +1,31 @@
...
 
 *Hi! My name is [Joana](https://github.com/joanafbrito) and I am a current Front End Engineering student at Turing School of Software Design. I am a career changer! I've  spent the last decade working to manage two small business, and grabbing the opportunity to move to another country.* 
 *Now, I'm spending my days learning code. As a woman I can contribuit to build a more balanced society, where people can be seen as equals in the software marketing and where I can build apps more inclusive and make a difference using this huge platform of the internet!*
 
+*Info about [Hayley](https://github.com/hayleyw7)🐈 * 
 
-*Info about Hayley 🐈 * 
-
-*Info about Matt 🎸   *
+*Info about [Matt](https://github.com/Matt-Roden) 🎸   *
```
Comment (Matt-Roden) [inline README.md:26]: Thanks for linking my github! I will write my summary about myself soon :)

**Review by Matt-Roden — APPROVED** [review]
Looks good!

**Pull request merged**